CHANGELOG.md

```diff
@@ -6,7 +6,9 @@
 
 ##### Enhancements
 
-* None.
+* Add `Request.failableSend()` that can handle SourceKitService crashes.
```

Comment:
There's also a breaking change in this PR: `SwiftDocs(file:arguments:)` now throws.